lib/core/services/sort-by-category.service.spec.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SortableByCategoryItem, SortByCategoryMapperService } from './sort-by-category.service';
+
+interface TestSortableByCategoryItem extends SortableByCategoryItem {
```

Comment:
are we extending it for a reason?

Reply:
This interface is only for presentation that we can use generic for our mapping service. I can remove it